.github/workflows/ci.yml

```diff
@@ -1,8 +1,8 @@
 name: 'CI Pipeline'
```

Comment:
In the JIRA I see the question "Do we want a nightly run of our automation?" - what did we decide for this? I don't see anything in this PR so guessing we're not doing it here. Otherwise if we are, it would be good if you could point it out. Thanks.

Reply:
I don't see a value in doing so - i'm unsure what it would give us that we don't currently have. I think justin agrees, but if you have an idea of something it would give us that we don't currently have i'm happy to hear it

Reply:
No I'm ok with not having it, given that we run pretty comprehensive testing with CI. Only if we wanted to do some lengthy extra testing nightly on top of what we do for CI, would I think it's worth it. Happy with your answer.